### ISSUE TITLE
Feature request: configurable mouse wheel speed in preview

### ISSUE THREAD

**Issue by carlfriedrich** [junegunn/fzf#3473]
- [x] I have read through the manual page (`man fzf`)
- [x] I have the latest version of fzf
- [x] I have searched through the existing issues

## Info

- OS
    - [x] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Shell
    - [x] bash
    - [ ] zsh
    - [ ] fish

## Problem / Steps to reproduce

I like the possibility to use mouse wheel scrolling in the preview pane. I do not use it a lot, though, because the scroll speed is too low for my taste (1 line at a time). Could we have the mouse wheel scroll speed configurable?

**Comment by junegunn** [comment]
Instead of adding a separate option to determine the amount of scrolling, I changed the implementation to use the event binding model. The solution is a bit more verbose, but it's much more flexible.

```sh
# Twice faster scrolling inside preview window
fzf --bind 'preview-scroll-up:preview-up+preview-up' \
    --bind 'preview-scroll-down:preview-down+preview-down' \
    --preview 'cat {}'
```

Please test it and let me know what you think.

**Comment by carlfriedrich** [comment]
@junegunn Great, thanks a lot for the quick implementation! When will this land in a release?

**Comment by junegunn** [comment]
I just released 0.43.0

**Comment by carlfriedrich** [comment]
@junegunn Great, thanks for the quick solution! Works like a charm! ❤
